src/components/ContentOpenWith/ContentOpenWith.js

```diff
@@ -42,28 +42,30 @@ type ExternalProps = {
 };
 
 type Props = {
-    /** Box File ID. */
-    fileId: string,
-    /** Application client name. */
-    clientName: string,
     /** Box API url. */
     apiHost: string,
-    /** Access token. */
-    token: Token,
     /** Class name applied to base component. */
     className: string,
+    /** Application client name. */
+    clientName: string,
+    /** Box File ID. */
+    fileId: string,
+    /** Determines positioning of menu dropdown */
+    isDropdownRightAligned: boolean,
```

Comment:
I do think that would be nicer for the consumer, but since this is a dropdown I really don't see any new values being added to this component. I'll try the enum approach quickly to see how that looks.